Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionLength", "type": "float32" },
+        { "name": "CancelType", "type": "ObjPhantomCancelCollisionSpawner::CancelType" }
+      ]
+    },
+    "ObjSetRigidBodySpawner": {
+      "fields": [
+        { "name": "Height", "type": "float32" },
+        { "name": "Width", "type": "float32" },
+        { "name": "Length", "type": "float32" },
+        { "name": "DefaultON", "type": "bool" },
+        { "name": "IsEffect", "type": "bool" },
+        { "name": "IsPlayerTerrain", "type": "bool" },
+        { "name": "m_IsSearchInsulateOnly", "type": "bool" },
+        { "name": "PhysicsType", "type": "ObjSetRigidBodySpawner::PhysicsType" }
+      ]
+    },
+    "ObjSwitchCollisionSpawner": {
+      "fields": [
+        { "name": "TargetList", "type": "array", "subtype": "object_reference" },
+        { "name": "EventTimer", "type": "float32" },
+        { "name": "EventType", "type": "ObjSwitchCollisionSpawner::EventType" },
+        { "name": "TargetList2", "type": "array", "subtype": "object_reference" },
+        { "name": "EventTimer2", "type": "float32" },
+        { "name": "EventType2", "type": "ObjSwitchCollisionSpawner::EventType2" },
+        { "name": "TargetList3", "type": "array", "subtype": "object_reference" },
+        { "name": "EventTimer3", "type": "float32" },
+        { "name": "EventType3", "type": "ObjSwitchCollisionSpawner::EventType3" },
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionDepth", "type": "float32" },
+        { "name": "EventArisingTimer", "type": "float32" },
+        { "name": "SwitchType", "type": "ObjSwitchCollisionSpawner::SwitchType" }
+      ]
+    },
+    "ObjFlowExSpawner": {
+      "fields": [
+        { "name": "EffectID", "type": "uint32" },
+        { "name": "EmitterCount", "type": "uint32" },
+        { "name": "GenerateRatio", "type": "uint32" },
+        { "name": "Speed", "type": "float32" },
+        { "name": "AngleX", "type": "float32" },
+        { "name": "AngleY", "type": "float32" },
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionLength", "type": "float32" }
+      ]
+    },
+    "ObjGismoSpawner": {
+      "fields": [
+        { "name": "ObjCategoryID", "type": "uint32" },
+        { "name": "ObjTypeID", "type": "uint32" }
+      ]
+    },
+    "ObjSetParticleSpawner": {
+      "fields": [
+        { "name": "EffectID", "type": "uint32" },
+        { "name": "OnTime", "type": "float32" },
+        { "name": "OffTime", "type": "float32" },
+        { "name": "ScaleX", "type": "float32" },
+        { "name": "ScaleY", "type": "float32" },
+        { "name": "ScaleZ", "type": "float32" },
+        { "name": "Trigger", "type": "ObjSetParticleSpawner::Trigger" }
+      ]
+    },
```

Comment:
Template parameters named based on SetParticle from Lost World and Forces and in-game functionality. Set data parser function (0x8012A5F0, NTSC-J Wii) used for additional help.